index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <a class="logo"> My Logo</a>
+    <img src="assets/Union.jpg" alt="menu-bar" />
+    <section class="intro-wrapper">
+      <h1 class="heading1">
+        I’m Linda Letaru<br />
+        Glad to see <br />you!
+      </h1>
+    </section>
+    <p class="content">
+      I’m a software developer! I can help you<br />
+      build a product , feature or website Look<br />
+      through some of my work and<br />experience! If you like what you see
+      and<br />
+      have a project you need coded, don’t<br />hestiate to contact me.
+    </p>
+    <div>
+      <br />
+      <p class="suggegiion-comment">LET’S CONNECT</p>
+      <br />
+      <div>
+        <h2 class="social media links">
```

Comment:
- [ ] [OPTIONAL] Please don't use ```hx``` to wrap element on your page, this is for the heading